manifest.json

```diff
@@ -38,19 +39,19 @@
   "content_scripts": [
     {
       "matches": ["<all_urls>"],
-      "js": ["content_scripts/fix-contentEditable.js"],
+      "js": ["vendor/browser-polyfill.js", "content_scripts/fix-contentEditable.js"],
```

Comment:
A separate injection could be better here.
Also, I think we could actually merge all of these into one, not sure why we haven't done that. cc @LEDfan